java-squid/src/main/java/org/sonar/java/JavaSquid.java

```diff
@@ -104,16 +98,17 @@ public JavaSquid(JavaConfiguration conf,
 
     //Bytecode scanner
     squidIndex = (SquidIndex) astScanner.getIndex();
-    bytecodeScanner = new BytecodeScanner(squidIndex, javaResourceLocator);
-    bytecodeScanner.accept(new DependenciesVisitor(graph));
+    bytecodeScanner = new BytecodeScanner(bytecodeContext);
+    DependenciesVisitor dependenciesVisitor = new DependenciesVisitor(bytecodeContext, graph);
+    bytecodeScanner.accept(dependenciesVisitor);
     for (CodeVisitor visitor : visitors) {
       bytecodeScanner.accept(visitor);
     }
 
   }
 
-  private static VisitorsBridge createVisitorBridge(Iterable<CodeVisitor> codeVisitors, List<File> classpath, JavaConfiguration conf, @Nullable SonarComponents sonarComponents) {
-    VisitorsBridge visitorsBridge = new VisitorsBridge(codeVisitors, classpath, sonarComponents);
+  private static InternalVisitorsBridge createVisitorBridge(Iterable<CodeVisitor> codeVisitors, List<File> classpath, JavaConfiguration conf, @Nullable SonarComponents sonarComponents) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 186 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00103)